db/warehouse/migrate/20230606184316_add_fa_amount_ranges.rb

```diff
@@ -0,0 +1,12 @@
+class AddFaAmountRanges < ActiveRecord::Migration[6.1]
+  def change
+    # new in 2024 spec
+    add_column :Services, :FAStartDate, :date, null: true
+    add_column :Services, :FAEndDate, :date, null: true
+
+    # add to CustomServices for custom financial assistance
+    add_column :CustomServices, :FAAmount, :float, null: true
```

Comment:
Is `float` the right thing to use? in the Services table the type is just "numeric" but I'm not sure how to achieve that or if its desirable